spec/syntax.md

```diff
@@ -677,16 +677,44 @@ unquoted-start = name-start / DIGIT / "."
 A **_<dfn>name</dfn>_** is an identifier for a _variable_ (prefixed with `$`),
 for a _function_ (prefixed with `:`, `+` or `-`),
 or for an _option_ (these have no prefix).
-The namespace for _names_ is based on XML's [Name](https://www.w3.org/TR/xml/#NT-Name),
-with the restriction that it MUST NOT start with `:`,
-as that would conflict with the _function_ start character.
+The namespace for _names_ is based on <cite>Namespaces in XML 1.0</cite>'s 
+[NCName](https://www.w3.org/TR/xml-names/#NT-NCName).
+This is different from XML's [Name](https://www.w3.org/TR/xml/#NT-Name)
+in that it MUST NOT start with `:`.
 Otherwise, the set of characters allowed in names is large.
 
+_Functions_ and _options_ MAY be preceded by a _namespace_ identifier
+which is separated from the body of the _name_ by a U+003A COLON `:`.
+Built-in _functions_ and _options_ do not have a _namespace_ identifier.
```

Comment:
What if an implementation extends a built-in function with additional options -- could those implementation-specific options be namespaced?

Reply:
Yes. In fact, we show this in the examples.

Reply:
I think it might be clearer for this sentence to say something like "Built-in _functions_ and standard _options_ to built-in functions..."

I'm not sure if "standard" is the right word there, but I mean the options specified in [the standard registry](https://github.com/unicode-org/message-format-wg/blob/main/spec/registry.xml) (if it's correct to call that the standard registry).